zuul_alpha/defaults.py

```diff
@@ -1,10 +1,17 @@
 import os
+from cryptography.hazmat.primitives import hashes
+from cryptography.hazmat.primitives.asymmetric import padding
 
 AWS_REGION = 'us-east-1'
 APP_ROOT = os.getenv('APP_ROOT', os.getcwd())
+DATA_DIR = 'zuul_data'
 DEFAULT_ENV = 'development'
 ENV_TAGS = [DEFAULT_ENV, 'staging', 'production']
 RSA_KEY_SIZE = 2048
 AES_KEY_SIZE = 256
 CIPHERTEXT_EXT = '.enc'
-ZUUL_DATA_DIR = os.path.join(APP_ROOT, 'zuul_data')
+
+RSA_PADDING = padding.OAEP(
```

Comment:
It seems like the crypto libraries with a big community following use the same set of ciphers. The built-in for Go uses this one: 

https://golang.org/pkg/crypto/rsa/

As far as configurability, if they aren't in the library, it should be possible, but not trivial to port your own as the Padding abstraction is there.